release-notes.txt

```diff
@@ -1,3 +1,232 @@
+
+network related issues
+----------------------
+#3075: nhdp: unnecessary microsecond precision
+    NHDP works with timer values of microsecond precision which is not required. Changing
+    to lower precision would save some memory.
+#3086: Max. packet length for AT86RF2XX
+    The size of the link-layer header is not dynamically calculated, but instead the maximum#
+    size is always assumed.
+#3970: RPL: Advertise DODAG only over the assigned interface
+     gnrc_rpl seems to multicast DIOs over all interfaces, though gnrc_rpl_init expects an
+     interface as parameter and sets the RPL-nodes multicast address only for that interface.
+#4048: potential racey memory leak
+     According to the packet buffer stats, flood-pinging a multicast destination may lead to a
+     memory leak due to a race condition. However, it seems to be a rare case and a
+     completely filled up packet buffer was not observed.
+#4462: IPHC/NHC broken between RIOT nodes
+
+native related issues
+---------------------
+#495: native not float safe
+    When the FPU is used when an asynchronous context switch occurs, either the
+    stack gets corrupted or a floating point exception occurs.
+#534: native debugging on osx fails
+    Using valgrind or gdb with a nativenet target in OSX leads to "the network"
+    being stuck (gdb) or the whole process being stuck (valgrind).
+#3341 and #3824: nativenet crashes when hammered
+    Flood-pinging a native instance from more than one host (either multiple threads on the
+    host system or multiple other native instances), leads to a SEGFAULT.
+#4608: tests/xtimer_usleep_until: unstable behaviour
+    The test starts to output "too large difference" and fails after a random period of time.
+
+other platform related issues
+-----------------------------
+#4560: make: clang is more pedantic than gcc
+    oonf_api is not building for clang on OS X
+#4583: cpp11: clang doesn't allow `mutex_t` to be used with `constexpr`
+    All cpp11-* tests fail for OS X with clang.
+
+other issues
+------------
+#2761: core: define default flags
+    If a thread is created without the corresponding flag  (CREATE_STACKTEST),
+    the ps command will yield wrong numbers for the  stack usage
+#2927: core: Automatically select the lowest possible LPM mode
+    Not all available low power modes (LPMs) are implemented for each platform and the
+    concept of how the LPM is chosen need some reconsideration
+#2967: Makefile.features: location is not relevant for all features
+    Provided features for the build system should be split up into a board and cpu specific
+    part
+#3109: periph/random: random_read should return unsigned int
+    The documentation of this function does not match corresponding implementation.
+#4488: Making the newlib thread-safe
+    When calling puts/printf after thread_create(), the CPU hangs. This seems to
+    be connected to the mutex used in the uart driver for syncing the DMA transfer
+    in uart_write().
```

Comment:
This also needs a more detailed description. Right now it sounds, that one cannot call puts()/printf() after thread_create(). If I understand the issue correctly, it will currently only fail for one platform and I have not fully understand what "calling ... after .." means. Time-wise after? Or in the next line(s)?